tests/ui/concrete-playback/mult-harnesses/expected

```diff
@@ -17,11 +17,11 @@ VERIFICATION:- FAILED
 Concrete playback
 ```
 #[test]
-fn kani_concrete_playback_harness1
+fn kani_concrete_playback_first_harness
     let concrete_vals: Vec<Vec<u8>> = vec![
         // 0
         vec![0]
     ];
-    kani::concrete_playback_run(concrete_vals, harness1);
+    kani::concrete_playback_run(concrete_vals, first::harness);
```

Comment:
Have you tried running this test? As far as I remember, the test is added to the same module has the harness, so I don't think the module should be added here.

Reply:
Fixed with https://github.com/model-checking/kani/pull/1955/commits/bcbd79e137b0bdb8f72b3469e0f1a05560817153

Although the CI passed before this fix, and maybe we need to adjust the CI to not pass.

Reply:
I totally agree. We need to add regression tests that actually runs the tests generated by  the concrete playback